tutorial/getting_started/install_dependencies.md

```diff
@@ -163,26 +141,36 @@ If you get an error at step 4 above, or can't launch `napari` after
 installation, you should try to delete your `napari-tutorial` environment, and
 follow the installation instructions below.
 
-1. Delete your `napari-tutorial` environment
+1. Delete your `image-analysis-23` environment
```

Comment:
The lines above this still mention `napari-tutorial`. But anyway I think all of this section can be removed.

Reply:
@kne42 indicated that this is still relevant in https://github.com/scipy-2023-image-analysis/tutorial/pull/5#discussion_r1248883819 so I'll keep it for now.